server/src/metadata/relation-metadata/relation-metadata.service.ts

```diff
@@ -168,16 +168,16 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
 
     const createdFieldMap = createdFields.reduce((acc, fieldMetadata) => {
       if (fieldMetadata.type === FieldMetadataType.RELATION) {
-        acc[fieldMetadata.objectMetadataId] = fieldMetadata;
+        acc[fieldMetadata.name] = fieldMetadata;
```

Comment:
can we pass the fieldMetadata.id instead? I feel the name is not robust enough. We could have two field with same name too

Reply:
@charlesBochet Actually we can't as we need to retrieve the fieldMetadata.id for `from` and `to` to store them inside the `relationMetadata`.
We can't base the code on the order of the declaration inside the `createMany` as Postgres doesn't guarantee the same order in the output of an `INSERT`.
Also name are unique and we can't have the same name multiple times on a same object :)